sale_order_invoicing_grouping_criteria/readme/INSTALL.rst

```diff
@@ -0,0 +1,2 @@
+This module only works in an updated 13.0 version after the 4th of March of 2020,
```

Comment:
install.rst file is no required any more

Reply:
Deleted.